helm/thingsboard/values.yaml

```diff
@@ -0,0 +1,285 @@
+  name: ""
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  host: chart-example.local
+  # Enable or disable tls, same host as above
+  tls: false
+  
+
+node:
+  # kind can be either Deployment or StatefulSet
+  kind: StatefulSet
+  loadDemo: false
+  replicaCount: 1
+  image:
+    repository: thingsboard/tb-node
+    # Overrides the global image values
+    #server: docker.io
+    #pullPolicy: Always
+    #tag: "X.Y.Z"
+  port:
+    http: 8080
+    rpc: 9090
+  podAnnotations: {}
+  podSecurityContext: {}
+    # fsGroup: 2000
+  securityContext: {}
+    # capabilities:
+    #   drop:
+    #   - ALL
+    # readOnlyRootFilesystem: true
+    # runAsNonRoot: true
+    # runAsUser: 1000
+  service:
+    type: ClusterIP
+    port: 8080
+  resources: {}
+    # We usually recommend not to specify default resources and to leave this as a conscious
+    # choice for the user. This also increases chances charts run on environments with little
+    # resources, such as Minikube. If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+    # requests:
+    #   cpu: 100m
+    #   memory: 128Mi
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    # targetMemoryUtilizationPercentage: 80
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+jsexecutor:
+  # kind can be either Deployment or StatefulSet
+  kind: Deployment
+  image:
+    repository: thingsboard/tb-js-executor
+    # Overrides the global image values
+    #server: docker.io
+    #pullPolicy: Always
+    #tag: "X.Y.Z"
+  replicaCount: 5
+  podAnnotations: {}
+  podSecurityContext: {}
+  securityContext: {}
+  resources: {}
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    # targetMemoryUtilizationPercentage: 80
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+mqtt:
+  # kind can be either Deployment or StatefulSet
+  kind: StatefulSet
+  image:
+    repository: thingsboard/tb-mqtt-transport
+    # Overrides the global image values
+    #server: docker.io
+    #pullPolicy: Always
+    #tag: "X.Y.Z"
+  replicaCount: 1
+  ssl:
+    enabled: false
+    secret: kubernetes-secret
+    cert: tls.crt
+    key: tls.key
+  port:
+    number: 8081
+    protocol: TCP
+    name: mqtt
+  timeout: 10000
+  podAnnotations: {}
+  podSecurityContext: {}
+  securityContext: {}
+  service:
+    type: ClusterIP
+    port: 8883
+  resources: {}
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    # targetMemoryUtilizationPercentage: 80
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+http:
+  # kind can be either Deployment or StatefulSet
+  kind: StatefulSet
```

Comment:
no way? This should default to Deployment...

Reply:
following upstream installation. Luckily I'm allowing us to replace in our environments